Symfony/CS/Fixer/PSR2/MethodArgumentSpaceFixer.php

```diff
@@ -29,27 +29,43 @@ public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
 
-        for ($index = $tokens->count() - 1; $index >= 0; --$index) {
+        for ($index = $tokens->count() - 1; 0 <= $index; --$index) {
```

Comment:
i took the solution that @keradus said me from strcitparamsfixer. 

There is no specific reason for one or another version,